W-14476576 DC sync not req duke (includes W-14476577,  W-14476579,  W-14776338) (#263)
mulesoft/docs-code-builder: modules/ROOT/pages/start-acb.adoc

.Click for more information:
[%collapsible]
====

> To save your work, you must either deploy your Mule projects to CloudHub or commit them to a source control management (SCM) system.

Reviewer: AFAIK Deploying Mule projects to CloudHub does not save your work since it only deploys the artifact

Author: thx, done

> To save your work, you must either deploy your Mule projects to CloudHub or commit them to a source control management (SCM) system.

Author: ```suggestion
To save your work, commit your project files to a source control management (SCM) system. See xref:start-scm.adoc[].
```

> To save your work, you must either deploy your Mule projects to CloudHub, or commit them to a source control management (SCM) system.

Author: ```suggestion
To save your work, you must either deploy your Mule projects to CloudHub or commit them to a source control management (SCM) system.
```

In the desktop IDE, you can change your Anypoint Platform control plane to the US or EU cloud. US cloud is the default. Anypoint Platform instances do not share user accounts, projects, or other data. For more information, see xref:index.adoc#us-eu-clouds[US and EU Cloud Hosts]. 

> //TODO (FEB): COULD THIS ISSUE STILL OCCUR IF THEY ARE CONFIGURED TO SYNC WITH DC?

Author: ```suggestion
```

> . If an API specification is open in the Explorer view of the IDE, close the project to avoid a potential synchronization error when you switch clouds. 

Reviewer: Not sure what synchronization errors we are referring to in this case

Author: TODO: ask Marina

Author: response is " I think apid team has done something around github sync but regarding the setup of your own github account, I think that if you switch between clouds, meaning EU and US, you will have sync issues. I would keep that in the documentation"

Author: done, keeping step

> //TODO (FEB): COULD THIS ISSUE STILL OCCUR IF THEY ARE CONFIGURED TO SYNC WITH DC?
> . If an API specification is open in the Explorer view of the IDE, close the project to avoid a potential synchronization error when you switch clouds. 

Author: ```suggestion
```
waiting to hear if I should remove this

> 
> * In the desktop IDE, click *File* > *Close Folder*.
> * In the cloud IDE, click the image:icon-menu.png["",18,18] (menu) icon, and select *File* > *Close Folder*.

Author: ```suggestion
```
along with these bullets (see above)
